spec/factories/posts.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :post do
```

Comment:
FactoryBot can be configured to create the required association `user` every time it creates a `post` instance. This is done by specifying the association in this file. Prefer doing it this way to avoid repeating it each time in the spec.

[`OPTIONAL`]

Reply:
Done